### PR TITLE
Update index.js

### PR DESCRIPTION
**Pull request by shoaib1991**
Updating user Routes

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -2,4 +2,7 @@ const app = require('express')();
 app.get('/', (req, res) => {
     res.send('Welcome');
 });
-app.listen(3000);
\ No newline at end of file
+app.get('/user', (req, res) = {
```
Comment (shoaibbhimani) [inline index.js]: Why did you use Arrow functions

Comment (shoaib1991) [reply index.js]: Aise Hi

Comment (shoaibbhimani) [reply index.js]: Please change it

**Comment by shoaibbhimani** [comment]
User Route Done

**Pull request merged**